modules/ipi-install-configuring-host-network-interfaces-in-the-install-config.yaml-file.adoc

```diff
@@ -6,64 +6,166 @@
+  - destination: 0.0.0.0/0
+    next-hop-address: <next_hop_ip_address> <1>
+    next-hop-interface: <next_hop_nic1_name> <1>
+----
++
+<1> Replace `<nic1_name>`, `<ip_address>`, `<dns_ip_address>`, `<next_hop_ip_address>` and `<next_hop_nic1_name>` with appropriate values.
 
-The most common use case for this functionality is to specify a static IP address on the `baremetal` network, but you can also configure other networks such as a storage network. This functionality will also support other NMState features such as VLAN, VXLAN, bridges, bonds, routes, MTU, and DNS resolver settings.
+.. Test the configuration file by running the following command:
++
+[source,terminal]
+----
+$ nmstatectl gc <nmstate_yaml_file>
+----
++
+Replace `<nmstate_yaml_file>` with the configuration file name.
 
+. Use the `networkConfig` configuration setting by adding the NMState configuration to hosts within the `install-config.yaml` file:
++
+[source,yaml]
+----
+    hosts:
+      - name: openshift-master-0
+        role: master
+        bmc:
+          address: redfish+http://<out_of_band_ip>/redfish/v1/Systems/
+          username: <user>
+          password: <password>
+          disableCertificateVerification: null
+        bootMACAddress: <NIC1_mac_address>
+        bootMode: UEFI
+        rootDeviceHints:
+          deviceName: "/dev/sda"
+        networkConfig: <1>
+          interfaces:
+          - name: <nic1_name> <2>
+            type: ethernet
+            state: up
+            ipv4:
+              address:
+              - ip: <ip_address> <2>
+                prefix-length: 24
+              enabled: true
+          dns-resolver:
+            config:
+              server:
+              - <dns_ip_address> <2>
+          routes:
+            config:
+            - destination: 0.0.0.0/0
+              next-hop-address: <next_hop_ip_address> <2>
+              next-hop-interface: <next_hop_nic1_name> <2>
+----
+<1> Add the NMState YAML syntax to configure the host interfaces.
++
+<2> Replace `<nic1_name>`, `<ip_address>`, `<dns_ip_address>`, `<next_hop_ip_address>` and `<next_hop_nic1_name>` with appropriate values.
++
 [IMPORTANT]
 ====
-Once deployed, you cannot modify the `networkConfig` configuration setting of `install-config.yaml` file to make changes to the host network interface. Use the Kubernetes NMState Operator to make changes to the host network interface after deployment.
+After deploying the cluster, you cannot modify the `networkConfig` configuration setting of `install-config.yaml` file to make changes to the host network interface. Use the Kubernetes NMState Operator to make changes to the host network interface after deployment.
 ====
+
+. If you are deploying {product-title} without a DHCP server on the `baremetal` network, configure a static IP address for the bootstrap VM using Ignition:
+
+.. Generate the manifests:
++
+[source,terminal]
+----
+$ ./openshift-baremetal-install --dir ~/clusterconfigs create manifests
+----
```

Comment:
Note this step is optional - it's only needed when modifying the installer generated manifests, otherwise you can just run `create ignition-configs` as below (then `create cluster`)